WalletWasabi.Gui/Global.cs

```diff
@@ -761,6 +761,17 @@ public async Task DisposeAsync()
 					Logger.LogError($"Error during the disposal of {nameof(SingleInstanceChecker)}: {ex}");
 				}
 
+				Logger.LogDebug($"Step: {nameof(SingleInstanceChecker)}.", nameof(Global));
```

Comment:
```suggestion
				Logger.LogDebug($"Step: {nameof(BitcoinStore)}.", nameof(Global));
```

Reply:
Hmm, general question: Should `BitcoinStore` possibly be disposed BEFORE `SingleInstanceChecker`? I understand that `SingleInstanceChecker` is one way how to prevent data corruption so I would expect that `SingleInstanceChecker` would be disposed as the very last thing to ensure safety of all stores/whatever.

Reply:
Yes, that is right. Fixed.